src/Icon/index.js

```diff
@@ -21,7 +21,7 @@ import Search from './Search'
 import Success from './Success'
 import User from './User'
 import Warning from './Warning'
-import ExternalLink from './ExternalLink'
+import IconLink from './IconLink'
```

Comment:
I the icon should named just `Link`, instead `IconLink` (the other icons do not include the term `Icon` in their names).